packages/amplify-category-geo/src/commands/geo.ts

```diff
@@ -0,0 +1,43 @@
+import { category } from '../constants';
+
+module.exports = {
```

Comment:
I realize this is just copying the format found in several other plugins, but we should switch to using ES6 export syntax. ie:
```js
export const name = category;
export const run = async () => {...}
```

(this applies to all the other command handlers in this PR as well)